Dockerfile

```diff
@@ -68,15 +38,37 @@ ENV DEFAULT_PLUGINS_REPO=sensu-plugins \
     EXTENSION_DIR=/etc/sensu/extensions \
     PLUGINS_DIR=/etc/sensu/plugins \
     HANDLERS_DIR=/etc/sensu/handlers \
-
     #Config for gathering host metrics
     HOST_DEV_DIR=/dev \
     HOST_PROC_DIR=/proc \
     HOST_SYS_DIR=/sys
-
-RUN mkdir -p $CONFIG_DIR $CHECK_DIR $EXTENSION_DIR $PLUGINS_DIR $HANDLERS_DIR
-
+RUN set -x \
+    && apt-get update \
+    && apt-get install -y --no-install-recommends \
+      curl \
+      ca-certificates \
+      apt-transport-https \
+      gnupg2 \
+      ${BUILD_DEPS} \
+    && curl -s https://sensu.global.ssl.fastly.net/apt/pubkey.gpg | apt-key add - \
+    && echo "deb     https://sensu.global.ssl.fastly.net/apt stretch main" > /etc/apt/sources.list.d/sensu.list \
+    && apt-get update \
+    && apt-get install -y sensu=${SENSU_VERSION} \
+    && gem install --no-document \
+        yaml2json \
+        eventmachine \
+    && curl -Ls https://github.com/Yelp/dumb-init/releases/download/v${DUMB_INIT_VERSION}/dumb-init_${DUMB_INIT_VERSION}_amd64.deb > dumb-init.deb \
+    && dpkg -i dumb-init.deb \
+    && rm dumb-init.deb \
+    && curl -Ls https://github.com/arschles/envtpl/releases/download/${ENVTPL_VERSION}/envtpl_linux_amd64 > /usr/local/bin/envtpl \
+    && chmod +x /usr/local/bin/envtpl \
+    && mkdir -p $CONFIG_DIR $CHECK_DIR $EXTENSION_DIR $PLUGINS_DIR $HANDLERS_DIR \
+    && rm -rf /opt/sensu/embedded/lib/ruby/gems/2.4.0/{cache,doc}/* \
+    && find /opt/sensu/embedded/lib/ruby/gems/ -name "*.o" -delete \
+    && apt-get purge --assume-yes ${BUILD_DEPS} \
+    && rm -rf /var/lib/apt/lists/*
+COPY templates /etc/sensu/templates
+COPY bin /bin/
 EXPOSE 4567
 VOLUME ["/etc/sensu/conf.d"]
-
-ENTRYPOINT ["/usr/bin/dumb-init", "--", "/bin/start"]
+CMD ["/usr/bin/dumb-init", "--", "/bin/start"]
```

Comment:
why was this changed to CMD from entrypoint?

Reply:
`ENTRYPOINT` won't let you `bash` in the container and debug some other stuff, `CMD` is more friendly.